.github/workflows/rust.yml

```diff
@@ -180,7 +180,7 @@ jobs:
       - name: Install Rust
         uses: actions-rs/toolchain@v1
         with:
-          toolchain: stable
+          toolchain: nightly
```

Comment:
@slowli, I suppose it makes sense because of: 
```
error[E0554]: `#![feature]` may not be used on the stable release channel
  --> src/lib.rs:82:21
   |
82 | #![cfg_attr(docsrs, feature(doc_cfg))]
   |    
```
???

Reply:
Yeah, that's my oversight. I've created a separate PR with a fix: #10.